src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -2,6 +2,40 @@
 
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        String[] dateFromParts = dateFrom.split("\\.");
+        String[] dateToParts = dateTo.split("\\.");
+        int fromDay = Integer.parseInt(dateFromParts[0]);
+        int fromMonth = Integer.parseInt(dateFromParts[1]);
+        int fromYear = Integer.parseInt(dateFromParts[2]);
+        int toDay = Integer.parseInt(dateToParts[0]);
+        int toMonth = Integer.parseInt(dateToParts[1]);
+        int toYear = Integer.parseInt(dateToParts[2]);
+        int startDateValue = fromYear * 10000 + fromMonth * 100 + fromDay;
+        int endDateValue = toYear * 10000 + toMonth * 100 + toDay;
```

Comment:
java has ready features for parsing string data such as "26.04.2019", google about. no need to invent bicycles :)
```suggestion
```